services/app-api/forms/mcpar.json

```diff
@@ -3974,7 +3978,7 @@
               "info": [
                 {
                   "type": "p",
-                  "content": "If ILOSs are authorized for this program, report for each plan: if the plan offered any ILOS; if “Yes”, which ILOS the plan offered; and utilization data for each ILOS offered. If the plan offered an ILOS during the reporting period but there was no utilization, check \"Yes\" that the ILOS was offered but enter \"0\" for utilization."
+                  "content": "If ILOSs are authorized for this program, report for each plan: if the plan offered any ILOS; if “Yes”, which ILOS the plan offered; and utilization data for each ILOS offered. If the plan offered an ILOS during the reporting period but there was no utilization, check that the ILOS was offered but enter \"0\" for utilization."
```

Comment:
Use smart quotes for `"0"`?

Reply:
We're thinking about refactoring `mcpar.json` and `mlr.json` to use smart quotes, but it'll be a larger lift outside of this PR.